profiles/templates/components/work-experience-form.html

```diff
@@ -5,20 +5,65 @@
   <h1 class="block text-violet-700 text-2xl font-bold mb-2">Work Experience</h1>
   <p class="block text-gray-700 text-sm mb-4">Please enter your work experience</p>
   <form method="post">
-    {{ work_experience_form.organization|as_crispy_field }}
-    {{ work_experience_form.position|as_crispy_field }}
+    {% csrf_token %}
+    <div class="md:w-[50%]">
+      {{ work_experience_form.organization|as_crispy_field }}
```

Comment:
Does this properly crispy them? Shouldn't it just be |crispy

Reply:
I am rendering the fields out individually so I can apply my own layout to them. I need to use as_crispy_field for this

Reply:
Oki